engines/native/system/variables_darwin_test.go

```diff
@@ -1,11 +0,0 @@
-package system
-
-// test variables
-var testGroup = "admin"
-var testCat = []string{"/bin/cat", "-"}
-var testTrue = []string{"/usr/bin/true"}
-var testFalse = []string{"/usr/bin/false"}
-var testPrintDir = []string{"/bin/pwd"}
-var testSleep = []string{"/bin/sleep", "5"}
-var testChildren = []string{"/bin/bash", "-c", "/bin/echo test $(sleep 20)"}
-var testGroups = []string{"staff", "admin"}
```

Comment:
OK, the linter didn't respect the build constraints! I see this is used when calling `go test -tags=system` (note we currently don't run these tests from our make file).